pkgs/applications/video/glaxnimate/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, zlib
+, potrace
+, ffmpeg
+, libarchive
+, python3
+, qtbase
+, qttools
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "glaxnimate";
+  version = "0.5.1";
+
+  src = fetchFromGitLab {
+    owner = "mattbas";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "G4ykcOvXXnVIQZUYpRIrALtDSsGqxMvDtcmobjjtlKw=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+  buildInputs = [
+    zlib
+    potrace
+    ffmpeg
+    # gst_all_1.gst-libav # Note that since gst-libav-1.6, libav is actually ffmpeg
```

```suggestion
    # Upstream asks for libav dependency, which is fulfilled by ffmpeg
    ffmpeg
```
(Or a more aptly worded comment)